prototype_source/graph_mode_dynamic_bert_tutorial.rst

```diff
@@ -0,0 +1,551 @@
+(prototype) Graph Mode Dynamic Quantization on BERT
+==============================================
+
+
+**Author**: `Supriya Rao <https://github.com/supriyar>`_
+
+Introduction
+------------
+
+This tutorial introduces the steps to do post training Dynamic Quantization with Graph Mode Quantization. Dynamic quantization converts a float model to a quantized model with static int8 data types for the weights and dynamic quantization for the activations. The activations are quantized dynamically (per batch) to int8 while the weights are statically quantized to int8. Graph Mode Quantization flow operates on the model graph and requires minimal user intervention to quantize the model. To be able to use graph mode, the float model needs to be either traced or scripted first.
+
+Advantages of graph mode quantization are:
+
+- In graph mode, we can inspect the code that is executed in forward function (e.g. aten function calls) and quantization is achieved by module and graph manipulations.
+- Simple quantization flow, minimal manual steps.
+- Unlocks the possibility of doing higher level optimizations like automatic precision selection.
+
+For additional details on Graph Mode Quantization please refer to the Graph Mode Static Quantization Tutorial.
```

Comment:
Suggestion: Add hyperlink for Graph Mode Static Quantization Tutorial

Reply:
The tutorial will be released along with this one. How do I link it in that case?